### ISSUE TITLE
下载400错误.  

### ISSUE THREAD

**Issue by gyf19** [jrnewell/goog-webfont-dl#13]
运行 ：goog-webfont-dl -a -f  Source+Sans+Pro -y 300,400,600,700,300italic,400italic,600italic

Downloading webfont formats: “ttf,eot,woff,woff2,svg” to folder “Source+Sans+
Pro”
http://fonts.googleapis.com/css?family=Source%2BSans%2BPro:300,400,600,700,300it
alic,400italic,600italic
Error: Bad response code: 400
    at Request._callback (C:\Users\Administrator\AppData\Roaming\npm\node_module
s\.goog-webfont-dl_npminstall\node_modules\.1.0.1@goog-webfont-dl\cssParser.js:2
9:23)
    at Request.self.callback (C:\Users\Administrator\AppData\Roaming\npm\node_mo
dules\.goog-webfont-dl_npminstall\node_modules\.2.53.0@request\request.js:344:22
)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (C:\Users\Administrator\AppData\Roaming\npm\node_modu
les\.goog-webfont-dl_npminstall\node_modules\.2.53.0@request\request.js:1239:14)

    at emitOne (events.js:82:20)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (C:\Users\Administrator\AppData\Roaming\npm\n
ode_modules\.goog-webfont-dl_npminstall\node_modules\.2.53.0@request\request.js:
1187:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickCallback (node.js:356:17)

**Comment by jrnewell** [comment]
Try this:
```shell
goog-webfont-dl -a -f "Source Sans Pro" -y 300,400,600,700,300italic,400italic,600italic
```

**Comment by jrnewell** [comment]
Fixed with 9d46dcbaabd376c22200cda0a7f463f54aea96d1
